frontend/src/bundles/common/components/header/components/notifications/components/notifications-bell.tsx

```diff
@@ -0,0 +1,35 @@
+import { BellIcon } from '@heroicons/react/24/solid';
+
+type Properties = {
+    count: number;
+    onClick: () => void;
+    showList: boolean;
+};
+
+const NotificationBell = ({
+    count,
+    onClick,
+    showList,
+}: Properties): JSX.Element => {
+    const animate = count > 0 && !showList;
+
+    return (
+        <button
+            className={`relative flex items-center justify-center ${animate && showList === false ? 'animate-pulse' : ''}`}
```

Comment:
we have helper for combining styles. That's not the only place, change everywhere